attributes.go

```diff
@@ -111,46 +111,48 @@ func (t AttrType) Value() uint16 {
 	return uint16(t)
 }
 
-var attrNames = map[AttrType]string{
```

Comment:
Why this change was made?
Global readonly maps are 100% ok.

Reply:
Changed to pass `gochecknoglobals`. Wrapped around a func to make it read-only.

Not sure if global map is read-only.

This also occurs for `methodName`

If you think var is better we can make a `//nolint` exception